test/mocha/10-api.js

```diff
@@ -297,6 +297,27 @@ describe('bedrock-account', () => {
         should.exist(err);
         err.name.should.equal('DuplicateError');
       });
+      it('should return error on duplicate email', async () => {
+        const email = '4c38d01c-d8fb-11ea-87d0-0242ac130003@example.com';
+        const newAccount = helpers.createAccount(email);
+        await brAccount.insert({
+          actor: null,
+          account: newAccount
+        });
+        // attempt to make another account with the same email
+        const newAccount2 = helpers.createAccount(email);
+        let err;
+        try {
+          await brAccount.insert({
+            actor: null,
+            account: newAccount2
+          });
+        } catch(e) {
+          err = e;
+        }
+        should.exist(err);
+        err.name.should.equal('DuplicateError');
```

Comment:
The DuplicateError message is "Duplicate account." right now. Not sure if we want to change that to something else.

Reply:
I think this is fine.